config/settings.yml

```diff
@@ -80,6 +80,8 @@
   :scan_via_host: true
   :use_vim_broker: true
   :use_vim_broker_ems: true
+:container_logging:
+  :common_logging_route: logging-kibana-ops
```

Comment:
This feels wrong as a setting in the general settings...feels like it should a per-provider settings or perhaps a special endpoint on each provider.